src/components/WaitList/WaitList.jsx

```diff
@@ -2,14 +2,41 @@ import React from 'react';
 import styles from './WaitList.module.scss'
 import clock from '../../utils/imgs/clock.png'
 import Reservation from '../Reservation';
+import { actionUpdateReservation } from '../../actions';
+import { connect } from "react-redux";
 
 const WaitList = (props) => {
     let {
         retailerName,
         waitList,
+        holdList,
         handlePlusPartySize,
     } = props;
 
+    const clearReservations = () => {
+        let waitLength = waitList.length;
+        let holdLength = holdList.length;
+
+        if (waitLength === 0 && holdLength === 0) {
+            return alert('No reservations to cancel.');
+        }
+        if (waitLength !== 0) {
+            waitList.forEach((res) => {
+                let data = { queueStatus: 'cancelled' }
+                props.dispatchUpdateReservation(data, res.id);
+            });
+            alert(`Wait list cleared ${waitLength} reservations.`)
+        };
+        if (holdLength !== 0) {
+            holdList.forEach((res) => {
+                let data = { queueStatus: 'cancelled' }
+                props.dispatchUpdateReservation(data, res.id);
+            })
+            alert(`Hold list cleared ${holdLength} reservations.`)
+        }
+        return
+    };
```

Comment:
Hey, I don't think this is good.  This requires the front end to fire off a request to the backend for every single reservation on the front, that could be a lot of calls.  I think we need to create a new backend route that will update the reservations based on what's saved in the retailer's reservation array.

Reply:
Lemme know if you wanna discuss and we can talk about what that might look like.

Reply:
Ok I'll look into updating multiple documents with one call